pkg/util/kubernetes/hpa/hpa_test.go

```diff
@@ -0,0 +1,153 @@
+package hpa
+
+import (
+	"testing"
+
+	"k8s.io/client-go/kubernetes/fake"
+	//fake2 "k8s.io/client-go/kubernetes/typed/core/v1/fake"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"encoding/json"
+	"fmt"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/api/autoscaling/v2beta1"
+	"k8s.io/api/core/v1"
+)
+
+// removeEntryFromConfigMap
+// ReadConfigMap
+
+func mockConfigMap(metricName string, labels map[string]string) *v1.ConfigMap {
```

Comment:
Nitpick: `newMockConfigMap`, `newFakeConfigMap` or `makeFakeConfigMap`. Same for the other functions that start with `mock*`.